app/javascript/components/workflows/professor/exams/new/editor/components/Submit.tsx

```diff
@@ -32,14 +32,14 @@ const Submit: React.FC<SubmitProps> = (props) => {
             if (res.updated === false) {
               alert({
                 variant: 'danger',
-                title: 'Exam not updated',
+                title: 'Exam version not updated.',
```

Comment:
Has version auto-save been implemented?  I think we may want these buttons to be changed to "Finished" and "Cancel".  We would stash the initial state of the exam upon loading the page, somewhere in the redux state; then Finished just takes you back to the exam admin page (because everything has been recently saved) and Cancel actually resets the state to the stashed value and saves it.

Reply:
We haven't done auto-save yet (#104).

note: we get 'reset' for free with redux-form, if we decide to use it for the exam editor (see the drag and drop forms)

Reply:
Well, if you wanna rewrite the editor to successfully use redux-form, go for it :)  That code could use a cleanup, for sure...